api/kyverno/v1/clusterpolicy_types.go

```diff
@@ -38,6 +38,8 @@ type ClusterPolicy struct {
 	// Status contains policy runtime data.
 	// +optional
 	Status PolicyStatus `json:"status,omitempty"`
+	// DeletionPropagationPolicy defines the deletion policy for the resources
```

Comment:
Why are you adding this to standard policies ?

Reply:
By adding the DeletionPropagationPolicy into standard policies, it ensures consistent behavior across various policy types, like users can expect the same set of features and capabilities when working with cleanup policies and cluster policies.